scripts/restore.sh

```diff
@@ -130,6 +130,7 @@ new_window() {
 	local dir="$4"
 	local pane_index="$5"
 	local pane_id="${session_name}:${window_number}.${pane_index}"
+   dir="${dir/#\~/$HOME}"
```

Comment:
Can you use tabs for indentation here?

Reply:
replaced with tabs, added .editorconfig to enforce "use tabs for spaces" rule ...